wit/deps/observe/world.wit

```diff
@@ -0,0 +1,8 @@
+package wasi:observe@0.2.0-draft;
+
+world imports {
+    import tracer;
+}
+
+// TODO(Reviewer): Should we make this an experimental wasi package or a Spin package
+// TODO(Reviewer): Would adding a metrics interface to this in a future version be a breaking change?
```

Comment:
@dicej does adding a new interface to a wold constitute a breaking change?

Reply:
It does not, no: additions to interfaces are semver-compatible. For now, that is not true for additions to:
- function parameters
- struct fields
- tuple entries
- variant and enum arms

Reply:
Just to clarify - it is not a breaking change from the perspective of the runtime, but it is from the perspective of the guest. In other words, if you add an import to a world, you can run old guests on an new runtime, but you cannot run new guests on an old runtime.